dev/integration_tests/ios_host_app/Host/DualFlutterViewController.m

```diff
@@ -28,18 +29,18 @@ - (void)viewDidLoad {
   stackView.layoutMarginsRelativeArrangement = YES;
   [self.view addSubview:stackView];
 
-  _topFlutterViewController = [[FlutterViewController alloc] init];
-  _bottomFlutterViewController= [[FlutterViewController alloc] init];
+  self.topFlutterViewController = [[FlutterViewController alloc] init];
+  self.bottomFlutterViewController= [[FlutterViewController alloc] init];
```

Comment:
I'm not sure if this ARC, this is a leak if it isn't.  It would be nice to assert arc or not at the top of the file.